From 227fee519e05d9d179d7ed9f1b11dd0f022afe9e Mon Sep 17 00:00:00 2001
From: SavantOS Release <actions@users.noreply.github.com>
Date: Fri, 11 Sep 2026 22:55:23 +0000
Subject: [PATCH] Refresh the guest package lock

archlinux-keyring 20260902-1 -> 20260909-1
cryptsetup 2.8.7-1 -> 2.8.8-1
enchant 2.8.19-1 -> 2.8.21-1
fcitx5 5.1.21-1 -> 5.1.22-1
fcitx5-qt 5.1.14-2 -> 5.1.15-1
foot 1.28.0-1 -> 1.28.0-2
fuse-common 3.18.2-1 -> 3.18.3-1
fuse3 3.18.2-1 -> 3.18.3-1
gnutls 3.8.13-2 -> 3.8.13-3
gpu-screen-recorder 6.1.0-1 -> 6.1.1-1
gst-plugins-bad-libs 1.28.6-3 -> 1.28.7-1
gst-plugins-base-libs 1.28.6-3 -> 1.28.7-1
gstreamer 1.28.6-3 -> 1.28.7-1
gtk-update-icon-cache 1:4.22.4-1 -> 1:4.22.5-1
gtk4 1:4.22.4-1 -> 1:4.22.5-1
gum 2.0.0-1 -> 2.0.1-1
hwdata 0.410-1 -> 0.411-1
hyprtoolkit 0.5.4-5 -> 0.5.4-6
hyprutils 0.14.1-1 -> 0.14.2-1
leancrypto 1.8.0-1 -> 1.9.0-1
libcap-ng 0.9.5-1 -> 0.9.6-1
libheif 1.23.3-1 -> 1.23.4-1
libnautilus-extension 50.3-1 -> 50.3.1-1
libpcap 1.10.6-1 -> 1.10.7-1
libwacom 2.19.1-1 -> 2.20.0-1
linux 7.2.3.arch1-2 -> 7.2.4.arch1-2
luajit 2.1.1788460057+24c20c9-1 -> 2.1.1788856981+c6ffc14-1
nautilus 50.3-1 -> 50.3.1-1
orc 0.4.43-1 -> 0.4.44-1
pkgconf 3.0.6-1 -> 3.0.7-1
plocate 1.1.24-2 -> 1.1.25-1
systemd 261.2-1 -> 261.3-1
systemd-libs 261.2-1 -> 261.3-1
systemd-resolvconf 261.2-1 -> 261.3-1
systemd-sysvcompat 261.2-1 -> 261.3-1
tensaku 0.28.0-1 -> 0.29.0-1
xz 5.8.3-1 -> 5.8.4-1
yyjson 0.12.0-1 -> 0.13.0-1
numactl 2.0.19-1 (new)
---
 guest/packages.lock.json | 77 ++++++++++++++++++++--------------------
 1 file changed, 39 insertions(+), 38 deletions(-)

diff --git a/guest/packages.lock.json b/guest/packages.lock.json
index 48156e9..b5dae6a 100644
--- a/guest/packages.lock.json
+++ b/guest/packages.lock.json
@@ -14,7 +14,7 @@
     "aom": "3.15.0-1",
     "appstream": "1.2.0-1",
     "aquamarine": "0.15.0-2",
-    "archlinux-keyring": "20260902-1",
+    "archlinux-keyring": "20260909-1",
     "at-spi2-core": "2.60.6-1",
     "attr": "2.6.0-1",
     "audit": "4.2.1-1",
@@ -45,7 +45,7 @@
     "coreutils": "9.11-2",
     "cpio": "2.15-3",
     "cpptrace": "1.0.4-2",
-    "cryptsetup": "2.8.7-1",
+    "cryptsetup": "2.8.8-1",
     "curl": "8.22.0-1",
     "dav1d": "1.5.4-1",
     "db5.3": "5.3.28-7",
@@ -64,7 +64,7 @@
     "duktape": "2.7.0-7",
     "e2fsprogs": "1.47.4-1",
     "elfutils": "0.196-1",
-    "enchant": "2.8.19-1",
+    "enchant": "2.8.21-1",
     "exempi": "2.6.6-3",
     "exiv2": "0.28.9-1",
     "expat": "2.8.4-1",
@@ -72,9 +72,9 @@
     "fakeroot": "1:1.37.2-3",
     "fastfetch": "2.68.1-1",
     "fcft": "3.3.3-1",
-    "fcitx5": "5.1.21-1",
+    "fcitx5": "5.1.22-1",
     "fcitx5-gtk": "5.1.7-1",
-    "fcitx5-qt": "5.1.14-2",
+    "fcitx5-qt": "5.1.15-1",
     "fd": "10.5.0-3",
     "ffmpeg": "2:9.0.1-4",
     "fftw": "3.3.11-1",
@@ -84,11 +84,11 @@
     "flac": "1.5.0-1",
     "flex": "2.6.4-6",
     "fontconfig": "2:2.18.3-2",
-    "foot": "1.28.0-1",
+    "foot": "1.28.0-2",
     "freetype2": "2.14.3-1",
     "fribidi": "1.0.16-2",
-    "fuse-common": "3.18.2-1",
-    "fuse3": "3.18.2-1",
+    "fuse-common": "3.18.3-1",
+    "fuse3": "3.18.3-1",
     "fzf": "0.74.3-1",
     "gawk": "5.4.1-1",
     "gc": "8.2.12-1",
@@ -119,13 +119,13 @@
     "gnome-keyring": "1:50.0-1",
     "gnulib-l10n": "20241231-1",
     "gnupg": "2.4.9-3",
-    "gnutls": "3.8.13-2",
+    "gnutls": "3.8.13-3",
     "gobject-introspection-runtime": "1.86.0-2",
     "gperftools": "2.18.1-1",
     "gpgme": "2.2.0-1",
     "gpgmepp": "2.2.0-1",
     "gpm": "1.20.7.r38.ge82d1a6-6",
-    "gpu-screen-recorder": "6.1.0-1",
+    "gpu-screen-recorder": "6.1.1-1",
     "graphene": "1.10.8-2",
     "graphite": "1:1.3.15-1",
     "grep": "3.12-2",
@@ -135,16 +135,16 @@
     "gsettings-system-schemas": "50.1-1",
     "gsm": "1.0.24-1",
     "gssdp": "1.6.6-1",
-    "gst-plugins-bad-libs": "1.28.6-3",
-    "gst-plugins-base-libs": "1.28.6-3",
-    "gstreamer": "1.28.6-3",
+    "gst-plugins-bad-libs": "1.28.7-1",
+    "gst-plugins-base-libs": "1.28.7-1",
+    "gstreamer": "1.28.7-1",
     "gtest": "1.18.0-1",
-    "gtk-update-icon-cache": "1:4.22.4-1",
+    "gtk-update-icon-cache": "1:4.22.5-1",
     "gtk3": "1:3.24.52-1",
-    "gtk4": "1:4.22.4-1",
+    "gtk4": "1:4.22.5-1",
     "gtk4-layer-shell": "1.3.0-1",
     "guile": "3.0.11-1",
-    "gum": "2.0.0-1",
+    "gum": "2.0.1-1",
     "gupnp": "1:1.6.10-1",
     "gupnp-dlna": "0.12.0-5",
     "gupnp-igd": "1.6.0-2",
@@ -156,7 +156,7 @@
     "hicolor-icon-theme": "0.18-1",
     "hidapi": "0.15.0-1",
     "highway": "1.4.0-1",
-    "hwdata": "0.410-1",
+    "hwdata": "0.411-1",
     "hwloc": "2.14.0-1",
     "hyprcursor": "0.1.13-7",
     "hyprgraphics": "0.5.1-4",
@@ -167,8 +167,8 @@
     "hyprlang": "0.6.8-5",
     "hyprpicker": "0.4.7-4",
     "hyprsunset": "0.4.0-3",
-    "hyprtoolkit": "0.5.4-5",
-    "hyprutils": "0.14.1-1",
+    "hyprtoolkit": "0.5.4-6",
+    "hyprutils": "0.14.2-1",
     "hyprwayland-scanner": "0.4.6-1",
     "hyprwire": "0.3.1-3",
     "iana-etc": "20260617-1",
@@ -196,7 +196,7 @@
     "l-smash": "2.14.5-4",
     "lame": "4.0-1",
     "lcms2": "2.19.1-1",
-    "leancrypto": "1.8.0-1",
+    "leancrypto": "1.9.0-1",
     "leptonica": "1.87.0-2",
     "less": "1:704-1",
     "libadwaita": "1:1.9.3-1",
@@ -226,7 +226,7 @@
     "libbsd": "0.12.2-2",
     "libbytesize": "2.12-3",
     "libcap": "2.78-1",
-    "libcap-ng": "0.9.5-1",
+    "libcap-ng": "0.9.6-1",
     "libcdio": "2.4.0-1",
     "libcdio-paranoia": "10.2+2.0.2-3",
     "libcgif": "0.5.4-1",
@@ -269,7 +269,7 @@
     "libgsf": "1.14.58-1",
     "libgudev": "238-3",
     "libgxps": "0.3.2-5",
-    "libheif": "1.23.3-1",
+    "libheif": "1.23.4-1",
     "libhwasan": "16.2.1+r23+gd564253eb6c8-1",
     "libice": "1.1.2-1",
     "libidn2": "2.3.8-1",
@@ -298,7 +298,7 @@
     "libmodplug": "0.8.9.0-7",
     "libmpc": "1.4.1-1",
     "libmysofa": "1.3.5-1",
-    "libnautilus-extension": "50.3-1",
+    "libnautilus-extension": "50.3.1-1",
     "libndp": "1.9-1",
     "libnetfilter_conntrack": "1.1.1-1",
     "libnewt": "0.52.25-2",
@@ -320,7 +320,7 @@
     "libopenmpt": "0.8.9-1",
     "libosinfo": "1.12.0-3",
     "libp11-kit": "0.26.5-1",
-    "libpcap": "1.10.6-1",
+    "libpcap": "1.10.7-1",
     "libpciaccess": "0.19-1",
     "libpgm": "5.3.128-4",
     "libpipeline": "1.5.8-1",
@@ -380,7 +380,7 @@
     "libvpl": "2.17.0-1",
     "libvpx": "1.17.0-1",
     "libvterm": "0.3.3-2",
-    "libwacom": "2.19.1-1",
+    "libwacom": "2.20.0-1",
     "libwebp": "1.6.0-2",
     "libwireplumber": "0.5.17-1",
     "libx11": "1.8.13-1",
@@ -418,7 +418,7 @@
     "libzip": "1.11.4-1",
     "licenses": "20240728-1",
     "lilv": "0.28.0-1",
-    "linux": "7.2.3.arch1-2",
+    "linux": "7.2.4.arch1-2",
     "linux-api-headers": "7.2-1",
     "llhttp": "9.3.1-1",
     "llvm-libs": "22.1.8-2",
@@ -428,7 +428,7 @@
     "lua": "5.5.1-1",
     "lua51-lpeg": "1.1.0-5",
     "lua54": "5.4.9-1",
-    "luajit": "2.1.1788460057+24c20c9-1",
+    "luajit": "2.1.1788856981+c6ffc14-1",
     "lv2": "1.18.10-2",
     "lz4": "1:1.10.0-2",
     "lzo": "2.10-5",
@@ -450,7 +450,7 @@
     "mtdev": "1.1.7-1",
     "mujs": "1.3.9-1",
     "muparser": "2.3.5-2",
-    "nautilus": "50.3-1",
+    "nautilus": "50.3.1-1",
     "ncurses": "6.6-2",
     "neovim": "0.12.5-1",
     "nettle": "4.0-1",
@@ -462,6 +462,7 @@
     "npth": "1.8-1",
     "nspr": "4.40-1",
     "nss": "3.128-1",
+    "numactl": "2.0.19-1",
     "ocl-icd": "2.3.5-1",
     "omacalc": "0.2.2-1",
     "omacut": "0.4.0-1",
@@ -479,7 +480,7 @@
     "openssh": "10.5p1-1",
     "openssl": "3.6.4-1",
     "opus": "1.6.1-1",
-    "orc": "0.4.43-1",
+    "orc": "0.4.44-1",
     "osinfo-db": "20260812-1",
     "p11-kit": "0.26.5-1",
     "pacman": "7.1.0.r9.g54d9411-2",
@@ -504,8 +505,8 @@
     "pipewire-audio": "1:1.6.8-1",
     "pipewire-pulse": "1:1.6.8-1",
     "pixman": "0.46.4-1",
-    "pkgconf": "3.0.6-1",
-    "plocate": "1.1.24-2",
+    "pkgconf": "3.0.7-1",
+    "plocate": "1.1.25-1",
     "polkit": "127-3",
     "poppler": "26.08.0-1",
     "poppler-glib": "26.08.0-1",
@@ -563,12 +564,12 @@
     "starship": "1.26.0-1",
     "sudo": "1.9.17.p2-6",
     "svt-av1": "4.2.0-1",
-    "systemd": "261.2-1",
-    "systemd-libs": "261.2-1",
-    "systemd-resolvconf": "261.2-1",
-    "systemd-sysvcompat": "261.2-1",
+    "systemd": "261.3-1",
+    "systemd-libs": "261.3-1",
+    "systemd-resolvconf": "261.3-1",
+    "systemd-sysvcompat": "261.3-1",
     "tar": "1.35-5",
-    "tensaku": "0.28.0-1",
+    "tensaku": "0.29.0-1",
     "tesseract": "5.5.3-1",
     "tesseract-data-eng": "2:4.1.0-5",
     "tesseract-data-osd": "2:4.1.0-5",
@@ -648,10 +649,10 @@
     "xorgproto": "2025.1-1",
     "xvidcore": "1.3.7-4",
     "xxhash": "0.8.3-1",
-    "xz": "5.8.3-1",
+    "xz": "5.8.4-1",
     "yay": "13.0.1-1",
     "yoga": "3.2.1-1",
-    "yyjson": "0.12.0-1",
+    "yyjson": "0.13.0-1",
     "zbar": "0.23.93-7",
     "zeromq": "4.3.5-3",
     "zimg": "3.0.6-2",
-- 
2.55.0

